torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -433,7 +441,7 @@ void PythonArgParser::print_error(PyObject* args, PyObject* kwargs, PyObject* pa
 
   std::vector<std::string> options;
   for (auto& signature : signatures_) {
-    if (!signature.deprecated) {
```

Comment:
do we still need deprecated?  I don't see it being read in python_arg_parser anymore.

Reply:
Yeah. We don't currently issue warning, but we will after we merge tensor and variable.

Reply:
What's the reason for not issuing a warning now?

Reply:
Some of the new signatures are only implemented on Variable, so the "correct" way to call (e.g. baddbmm) on Variables and Tensors would differ. I'd rather wait until after we kill Tensor to start issuing warnings.